src/graphql/listSubmissions.ts

```diff
@@ -0,0 +1,53 @@
+import gql from 'graphql-tag';
+
+export const query = gql`
+  query listSubmissions($first: Int, $offset: Int, $orderBy: String, $sortDirection: String, $organization: String, $status: String) {
+    listSubmissions(first: $first, offset: $offset, orderBy: $orderBy, sortDirection: $sortDirection, organization: $organization, status: $status) {
+      total
+      submissions {
+        _id
+        name
+        submitterName
+        organization {
+          _id
+          name
+        }
+        dataCommons
+        studyAbbreviation
+        dbGaPID
+        status
+        concierge
+        createdAt
+        updatedAt
+      }
+    }
+  }
+`;
+
+export type Response = {
+  listSubmissions: {
+    total: number;
+    submissions: Omit<Submission, "submitterID"
+    | "modelVersion" | "bucketName" | "rootPath" | "history">[];
+  };
+};
+
+// type Submission {
```

Comment:
Please confirm if we need these comments for something or if they are an accident.